firstvoices/backend/serializers/category_serializers.py

```diff
@@ -36,8 +36,16 @@ class Meta(ChildCategoryListSerializer.Meta):
 
 
 class ParentCategoryFlatListSerializer(ChildCategoryListSerializer):
+    parent = serializers.UUIDField(source="parent_id", allow_null=True, read_only=True)
+    parentTitle = serializers.SerializerMethodField(
```

Comment:
[minor] for consistency convert this to snake case:

`parent_title` and it will be converted to camel case by drf